lib/origen_testers/vector_pipeline.rb

```diff
@@ -124,7 +124,7 @@ def yield_vector(vector, &block)
       else
```

Comment:
Is the same patch not required on line 123 also?

Reply:
Had to move cycle tracking further upstream